README.md

```diff
@@ -484,6 +484,14 @@ LevelDB is thread-safe but is **not** suitable for accessing with multiple proce
 
 See the <a href="https://github.com/level/levelup/wiki/Modules"><b>wiki</b></a> for some LevelUP extensions, including [multilevel](https://github.com/juliangruber/multilevel), that may help if you require a single store to be shared across processes.
 
+<a name="typings"></a>
+Typescript 
+----------
+
+LevelUP comes with typescript definitions that can automatically infer options based off a typed level-down implementation. 
```

Comment:
- "based off" => "based on"?
- "level-down implementation" => we commonly stylize this as `abstract-leveldown` implementation (with backticks)

Reply:
fixed 👍